pylintrc_tweaks

```diff
@@ -10,3 +10,4 @@ DISABLE+=
     django-not-configured,
     consider-using-with,
     useless-option-value,
+    superfluous-parens,
```

Comment:
the linter complained about 
```python
if not (lower_bound <= requested_price_cents <= upper_bound):
```
but in my not-so-humble opinion, parens actually make this (and other such statements) much easier to read.  I think this violation, in general, is misguided.